app/src/components/404/404.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import style from "./style.module.scss";
+import { NavLink } from "react-router-dom";
+import nf404 from '../../assets/img/404.png';
+
+class NotFound extends React.Component {
+  render() 
+  {
+    return(
+        <div className={style.container}>
+              <img src={nf404} alt="page not found" className={style.img}/>
+          <div className={style.text}>
+            Приносим свои извинения за предоставленные неудобаства, запрашиваемая вами страница не найдена или удалена.
+            Проверьте корректность вашего URL .
+          </div>
+          <div className={style.buttons}>
+            <NavLink className={style.button} onClick={this.props.history.goBack}>Назад     </NavLink>
```

Comment:
Use common buttons from antd
its helps our project looks same in the differents parts and pages

Reply:
First, I made the buttons from antd, then the links, and the links turned out better as for me ... Buttons are like buttons)